src/app/js/admin/parsing/ParsingResult.js

```diff
@@ -381,6 +387,31 @@ export const ParsingResultComponent = props => {
         );
     };
 
+    const MyLink = props => <Link to="/data/add" {...props} />;
```

Comment:
I don't understand why you create this component. I think that using standard Link as component of MUI button is enough. Am I wrong ?
And you put the route in two different places